addons/web/static/src/model/relational_model/utils.js

```diff
@@ -161,6 +162,7 @@ export function patchActiveFields(activeField, patch) {
     activeField.onChange = activeField.onChange || patch.onChange;
     activeField.forceSave = activeField.forceSave || patch.forceSave;
     activeField.isHandle = activeField.isHandle || patch.isHandle;
+    activeField.fieldsTofetch = activeField.fieldsTofetch || patch.fieldsTofetch;
```

Comment:
No need for yet another mecanism, we already have relatedFields. Only thing is, it is currently restricted to x2many fields (see in field.js, parseFieldNode), but you could extend it to many2ones. I had to do it for many2one_reference here https://github.com/odoo/odoo/pull/149906/commits/afaa93479d99a9a6b32f10207a94bb205fabc211#diff-048974380a7d10c4b17104de98210d5b310c50dfa161356eedd0880199094de2R239

Reply:
I'm going to look at this then! If I can avoid editing model.py, it will be a lot easier.